### PR TITLE
Migrate changeCSS() to CSS variable in utils/utils.jsx, ln. 701

### PR DESCRIPTION
**Pull request by zeirash**
Summary
Migrate changeCSS() function for .pinned_posts-button to CSS variable

Ticket Link
Fixes mattermost/mattermost-server#16029

### CODE REVIEW

**Comment by hanzei** [comment]
@zeirash Thanks for the PR! It seems like there are some unrelated commits in your branch. Should be base branch for this PR be `campaign/applytheme_center-channel-color`?

**Comment by zeirash** [comment]
Ah yes thank you for reminding. Base branch is updated @hanzei

File: sass/layout/_headers.scss [hunk 1]
```diff
@@ -180,6 +180,12 @@
     .disabled {
         color: #999999;
     }
+
+    .pinned-posts-button {
+        svg {
+            fill: rgba(var(--center-channel-color), 0.6);
```
Comment (asaadmahmood) [inline sass/layout/_headers.scss:186]: Please use `--center-channel-color-rgb`.

**Review by asaadmahmood — CHANGES_REQUESTED** [review]

File: sass/layout/_headers.scss [hunk 2]
```diff
@@ -180,6 +180,12 @@
     .disabled {
         color: #999999;
     }
+
+    .pinned-posts-button {
+        svg {
+            fill: rgba(var(--center-channel-color-rgb), 0.6);
+        }
+    }
```
Comment (deanwhillier) [inline sass/layout/_headers.scss:188]:
```suggestion
```

Comment (deanwhillier) [reply sass/layout/_headers.scss:188]: Apologies @zeirash, but it looks like the class `.pinned-posts-button` is actually no longer used. The channel header had some work done but it looks like this `changeCSS` line was inadvertently left behind.

@asaadmahmood, do you concur?

**Review by deanwhillier — CHANGES_REQUESTED** [review]

File: sass/layout/_headers.scss (hunk 2 above)
Comment (zeirash) [reply sass/layout/_headers.scss:188]: so does that mean issue #16029 is no longer valid @deanwhillier ?

Comment (asaadmahmood) [reply sass/layout/_headers.scss:188]: Probably. We can close this ticket then.

**Pull request closed without merging**